inst/qml/Descriptives.qml

```diff
@@ -292,7 +292,7 @@ Form
 			}
 			CheckBox
 			{
-				name: "addSmooth"
+				name: "regressionLine"
```

Comment:
as these options affect only the scatter plot, prefix the `scatterPlot` name:

```suggestion
				name: "scatterPlotRegressionLine"
```

Also for the other suboptions:
`graphTypeAbove` -> `scatterPlotGraphTypeAbove`
`graphTypeRight` -> `scatterPlotGraphTypeRight`
`regressionType` -> `scatterPlotRegressionLineType`